src/components/Book/book.css

```diff
@@ -1,33 +1,93 @@
-.title {
-  font-size: 20px;
-  font-weight: bold;
-  width: 85vw;
+.book {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-around;
+  width: 90vw;
+  margin: 1rem 1rem 0 3.5%;
+  padding: 1rem;
+  border: 1px solid #d2d2d2;
   border-radius: 4px;
-  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.category {
+  font-size: 13px;
+  text-transform: uppercase;
+  color: #c5c5c5;
+  font-weight: 700;
+}
```

Comment:
- [ ] Please 🙏🏽 use the styles assigned to the category in the original design to match with Zeplin design 👇🏽 
![image](https://github.com/Booze33/book-store/assets/58642949/bd8c651b-d1c0-4194-ac5b-b3efe49e4cf8)